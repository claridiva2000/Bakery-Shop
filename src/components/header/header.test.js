import React from 'react';
import renderer from 'react-test-renderer';

import Header from './header.component';

describe('Header component', () => {
  it('should render Header component', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
