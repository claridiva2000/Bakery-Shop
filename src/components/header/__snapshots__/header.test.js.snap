// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header component 1`] = `
<div
  className="Header"
>
  <div
    className="top-nav"
  >
    <div
      className="social"
    >
      <svg>
        user-alt-solid.svg
      </svg>
      <svg>
        facebook-square-brands.svg
      </svg>
      <svg>
        instagram-brands.svg
      </svg>
    </div>
    <div
      className="logo"
    >
      <div>
        <span>
          Swayzer's
        </span>
        <a
          href="/"
          onClick={[Function]}
        >
          <svg>
            logo.svg
          </svg>
        </a>
        <span>
          Sweets
        </span>
      </div>
      <span
        className="logo-text"
      >
        Specialty Cakes and Cookies
      </span>
    </div>
    <div
      className="cart-order"
    >
      <button
        className="custom-button"
      >
        Order Online
      </button>
      <svg>
        shopping-cart-solid.svg
      </svg>
    </div>
  </div>
  <nav>
    <div>
      <span>
        Swayzer's Sweets
      </span>
      <svg>
        bars-solid.svg
      </svg>
    </div>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/about"
      onClick={[Function]}
    >
      About Us
    </a>
    <a
      href="/shop"
      onClick={[Function]}
    >
      Shop
    </a>
    <a
      href="/contact"
      onClick={[Function]}
    >
      Contact Us
    </a>
  </nav>
</div>
`;
